sdk/eventhub/Azure.Messaging.EventHubs/tests/Core/EventDataTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="EventData" />
+    ///   class.
+    /// </summary>
+    ///
+    [TestFixture]
+    public class EventDataTests
+    {
+        /// <summary>
+        ///   Verifies functionality of the <see cref="EventData.BodyAsStream "/>
+        ///   property.
+        /// </summary>
+        ///
+        [Test]
+        public void BodyAsStreamReturnsTheBody()
+        {
+            var eventData = new EventData(new byte[] { 0x11, 0x22, 0x65, 0x78 });
+
+            using var bodyStream = new MemoryStream();
+            eventData.BodyAsStream.CopyTo(bodyStream);
```

Comment:
We're not properly disposing of the stream returned by `BodyAsStream`, are we? By copying its contents, we're just reading and writing them again into another stream, but the intermediate object still exists.

Same question applies to the other test.

Reply:
Good catch.   We're typically a bit sloppy with resources in the case of tests, since the scope is transient... but we should still be disposing.   I'll adjust.